apps-rendering/src/components/NewsletterSignup/PrivacyWording.tsx

```diff
@@ -0,0 +1,76 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import { neutral, text, textSans } from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import type { FC } from 'react';
+
+// ----- Component ----- //
+
+interface Props {
+	notUsingCaptcha?: boolean;
```

Comment:
Does this prop have to be optional? Can it just be a required `boolean`?

If it does have to be optional we have a convention for using the `Option` type and `maybeRender` in AR. This helps to avoid the complexities that can come with implicit truthiness and the `&&` operator. Happy to talk through how these work if you'd like?

Reply:
It doesn't have to be optional - I'll make it required. It probably best as a required prop as it should be a conscious, explicit decision to include the captcha wording or not.